mailpoet/lib/Newsletter/Renderer/Blocks/Button.php

```diff
@@ -35,7 +35,29 @@ public function render($element, $columnBaseWidth) {
                   </v:roundrect>
                   <![endif]-->
                   <!--[if !mso]><!-- -->
-                  <a class="mailpoet_button" href="' . EHelper::escapeHtmlLinkAttr($element['url']) . '" style="' . $styles . '"> ' . EHelper::escapeHtmlText($element['text']) . '</a>
+                  <table
+                    border="0"
+                    cellspacing="0"
+                    cellpadding="0"
+                    role="presentation"
+                    style="display:inline-block;border-collapse:separate;mso-table-lspace:0;mso-table-rspace:0;width:' . EHelper::escapeHtmlStyleAttr($element['styles']['block']['width']) . '"
+                    width="' . EHelper::escapeHtmlStyleAttr($element['styles']['block']['width']) . '"
```

Comment:
The wrapping table has a width calculated as button width - border, but it wraps the button with the border, so the inner button (td) overflows the wrapping table. It causes the whole button (if centered) to misalign a bit to the right. We could use the original width for the wrapping table.
Please see the screenshot.

Here, I have a centered button with a width set to 280px in the editor and a border of 10px.
![Screenshot 2024-07-23 at 9 42 07](https://github.com/user-attachments/assets/382fe957-3023-46c4-b4ba-80bf49c31f3e)

Reply:
Good catch. I am going to fix it.